gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -185,10 +185,10 @@ public ZoneInfo build() {
   }
 
   /**
-   * Returns a builder for {@code ZoneInfo} with an assigned {@code name}.
+   * Returns a ZoneInfo object with assigned {@code name}, {@code dnsName} and {@code description}.
    */
-  public static Builder builder(String name) {
-    return new BuilderImpl(name);
+  public static ZoneInfo of(String name, String dnsName, String description) {
```

Comment:
`description` is optional right? What if I don't want to provide one?
If you confirm description is optional (also what about `dnsName`) you could consider overloading `of` with:
```java
public static ZoneInfo of(String name, String dnsName) {
  return new BuilderImpl(name).dnsName(dnsName).build();
}
```